robotino_calibration/ros/src/camera_base_calibration_marker.cpp

```diff
@@ -216,20 +206,20 @@ bool CameraBaseCalibrationMarker::moveRobot(const calibration_utilities::RobotCo
 	//Avoid that robot moves, when there is an error with detecting the wall!
 
 	// move pan-tilt unit
-	std_msgs::Float64 msg;
-	msg.data = robot_configuration.pan_angle_;
-	calibration_interface_->assignNewCamaraPanAngle(msg);
-	//pan_controller_.publish(msg);
-	msg.data = robot_configuration.tilt_angle_;
-	//tilt_controller_.publish(msg);
-	calibration_interface_->assignNewCamaraTiltAngle(msg);
+	std_msgs::Float64MultiArray angles;
 	
+	angles.data.resize(2);
+	angles.data[0] = robot_configuration.pan_angle_;
```

Comment:
Why is robot_configuration not changed to contain a variable sized array of pan/tilt/etc. angles?